docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,214 @@
+---
+---
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Running Multiple Schedulers
+
+Kubernetes ships with a default scheduler that is described in [scheduler documentation](kube-scheduler.md). If the default scheduler shipped with Kubernetes does not suit your needs you can implement your own scheduler. Not just that, you can even run multiple schedulers simultaneously alongside the default scheduler and instruct Kubernetes what
+scheduler to use for each of your pods. Let's learn how to run multiple schedulers in Kubernetes with an example.
+
+Detailed description of how to implement a scheduler is outside the scope of this document. Please refer to the kube-scheduler implementation in [plugin/pkg/scheduler](https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/scheduler) in the Kubernetes source directory for a canonical example.
+
+### 1. Package the scheduler
+
+Package your scheduler binary into a container image. For the purposes of this example, let's just use kube-scheduler as our second scheduler as well. Clone the [Kubernetes source code from Github](https://github.com/kubernetes/kubernetes) and build the source.
+
+```sh
+$ git clone https://github.com/kubernetes/kubernetes.git
+$ cd kubernetes
+$ hack/build-go.sh
+```
+
+Copy the built kube-scheduler binary into the container image. Here is the `Dockerfile` to build the image:
+
+```Dockerfile
+FROM busybox
+ADD _output/local/go/bin/kube-scheduler /usr/local/bin/kube-scheduler
+```
+
+Save the file as `Dockerfile`, build the image and push it to a registry. This example pushes the image to [Google Container Registry (GCR)](https://cloud.google.com/container-registry/). For more details, please read the GCR [documentation](https://cloud.google.com/container-registry/docs/).
+
+```sh
+$ docker build -t my-kube-scheduler:1.0 .
+$ gcloud docker push gcr.io/my-gcp-project/my-kube-scheduler:1.0
```

Comment:
Pretty sure you need to tag the image first

docker build..
docker tag my-kube-scheduler:1.0  gcr.io/my-gcp-project/my-kube-scheduler:1.0
gcloud docker push...